### PR TITLE
Advfilter testing

### PR DESCRIPTION
**Pull request by DiegoTheExplorar**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/AdvFilterCommandTest.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
```
Comment (valhrd) [inline src/test/java/seedu/address/logic/commands/AdvFilterCommandTest.java:5]: TIL

File: src/test/java/seedu/address/logic/commands/AdvFilterCommandTest.java [hunk 2]
```diff
@@ -81,8 +83,64 @@ public void execute_tagValue_equalsHigh() {
...
+    public void execute_noContactsFound() {
+        AdvFilterCommand command = new AdvFilterCommand("priority", "=", "nonexistent");
+        CommandResult result = command.execute(model);
+
+        assertEquals(AdvFilterCommand.MESSAGE_NO_CONTACT_FOUND, result.getFeedbackToUser());
+        assertTrue(model.getFilteredPersonList().isEmpty());
+    }
+
+
+
```
Comment (valhrd) [inline src/test/java/seedu/address/logic/commands/AdvFilterCommandTest.java:145]: LGTM

File: src/test/java/seedu/address/logic/parser/AdvFilterCommandParserTest.java [hunk 3]
```diff
@@ -37,11 +40,14 @@ public void parse_validArgs_returnsFilterCommand() {
         assertParseFailure(parser, "t/friend =",
                 String.format(MESSAGE_INVALID_COMMAND_FORMAT, AdvFilterCommand.MESSAGE_USAGE));
 
-        //only tag name and tag value provided
+        // only tag name and tag value provided
```
Comment (valhrd) [inline src/test/java/seedu/address/logic/parser/AdvFilterCommandParserTest.java:43]: Nice find!

File: src/test/java/seedu/address/logic/parser/AdvFilterCommandParserTest.java [hunk 4]
```diff
@@ -53,9 +59,63 @@ public void parse_validArgs_returnsFilterCommand() {
...
+
+        // Test all fields null or empty
+        assertParseFailure(parser, "t/   ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, AdvFilterCommand.MESSAGE_USAGE));
+    }
+    @Test
+    public void parse_extraSpacesBetweenOperatorAndValue_returnsFilterCommand() {
+        AdvFilterCommand expectedCommand = new AdvFilterCommand("age", "=", "25");
+        assertParseSuccess(parser, "t/age =   25", expectedCommand);
+    }
```
Comment (valhrd) [inline src/test/java/seedu/address/logic/parser/AdvFilterCommandParserTest.java:118]: Excellent implementation!

**Comment by valhrd** [comment]
LGTM

File: src/test/java/seedu/address/logic/commands/AdvFilterCommandTest.java (hunk 2 above)
Comment (RyanWarwick) [reply src/test/java/seedu/address/logic/commands/AdvFilterCommandTest.java:145]: LVGTM

File: src/test/java/seedu/address/logic/parser/AdvFilterCommandParserTest.java (hunk 4 above)
Comment (RyanWarwick) [reply src/test/java/seedu/address/logic/parser/AdvFilterCommandParserTest.java:118]: Appreciate the extensive test cases for various operators

**Pull request merged**